lib/internal/errors.js

```diff
@@ -858,6 +859,22 @@ const genericNodeError = hideStackFrames(function genericNodeError(message, erro
   return err;
 });
 
+ObjectDefineProperty(AggregateErrorPrototype, SymbolFor('nodejs.util.inspect.custom'), {
```

Comment:
is there a way this can work for aggregate errors from any realms? maybe in util.inspect check for name is `AggregateError` and `errors` property?

Reply:
Yes, checking for `.errors` in `formatError()` looks much more appropriate. Updated.

I don't think it's worth to check `.name` as well, mainly because there might be custom errors extending AggregateErrors.